lib/rspec/mocks/error_generator.rb

```diff
@@ -1,4 +1,5 @@
 RSpec::Support.require_rspec_support "object_formatter"
+RSpec::Support.require_rspec_support 'ruby_features'
```

Comment:
See other comment, now if we need this again elsewhere we'll have to remember to require it / move it back.

Reply:
That's what I did, went through all our files and added/removed it to only be required where used.